src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SourceTelemetry.cs

```diff
@@ -75,6 +76,11 @@ private enum HttpStyle
                     {
                         if (source.IsHttp)
                         {
+                            if (source.IsHttps)
+                            {
```

Comment:
Adding IsHttpsV2 and IsHttpsV3 could be useful, because we can have more knowledge on which remote feeds are used with HTTPS